test/setup/main.tf

```diff
@@ -14,20 +14,54 @@
  * limitations under the License.
  */
 
+data "terraform_remote_state" "org" {
+  backend = "gcs"
+  config = {
+    bucket = "cft-infra-test-tfstate"
+    prefix = "state/org"
+  }
+}
+
+data "google_organization" "org" {
+  organization = "organizations/${var.org_id}"
+}
+
+resource "google_organization_iam_member" "sa_org" {
+  for_each = toset([
+    "roles/resourcemanager.organizationViewer"
+  ])
+
+  org_id = var.org_id
+  role   = each.value
+  member = "serviceAccount:${data.terraform_remote_state.org.outputs.ci_gsuite_sa_email}"
+}
+
+// Create a temporary project to host group member service accounts to pass to the examples.
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 8.0"
+  version = "~> 9.0"
 
-  name              = "ci-group"
-  random_project_id = "true"
-  org_id            = var.org_id
-  folder_id         = var.folder_id
-  billing_account   = var.billing_account
+  name                 = "ci-group"
+  random_project_id    = "true"
+  org_id               = var.org_id
+  folder_id            = var.folder_id
+  billing_account      = var.billing_account
   skip_gcloud_download = true
 
   activate_apis = [
     "cloudresourcemanager.googleapis.com",
-    "storage-api.googleapis.com",
     "serviceusage.googleapis.com"
   ]
 }
+
+resource "google_service_account" "member" {
```

Comment:
Can we move the creation of the service accounts to inside the example itself? There's no need to pass them all the way from setup/ to the example (in general we're trying to limit the logic in setup/).

Reply:
I actually added them in the example. But because we are reusing the `ci-gsuite-sa-project` project and the gsuite SA in this case. So the service accounts inside the example itself will be created under the persistent `ci-gsuite-sa-project` project. And during testing, when tests fail in the middle, those testing example service accounts will not be destroyed in the `ci-gsuite-sa-project` by the test itself. They need to be manually deleted. So after discussing with @bharathkkb, we think it's better to create those service accounts in a temporary testing project created during setup, so even if tests fail in the middle and this testing project did not get destroyed by kitchen, they will be deleted by another separate clean up script.

Reply:
@xingao267 IIUC what @morgante meant was to use created project id from setup in examples and then within example have 
```hcl
resource "google_service_account" "member" {
  project      = var.sa_project_id
...
```
As `google_service_account` is being created in `sa_project_id`, this will get cleaned up.

My thoughts with this were we would have to depend on a non computed value but as you mentioned it still enforces dependency https://github.com/terraform-google-modules/terraform-google-group/pull/1#discussion_r528010019 so it should be okay.

Reply:
Done! Thanks!